cookbooks/passenger5/recipes/install.rb

```diff
@@ -12,7 +12,7 @@
 # Install gems required by Passenger standalone
 ruby_block "gems to install" do
```

Comment:
Slightly off-topic, but why isn't this an `execute` resource?

Reply:
I have no idea. It looks like there's no reason here to use a ruby_block.